backend/datasets/generate_gp_online.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import numpy as np
+import os
+
+# ------------------
+# Read in the data
+# ------------------
+
+GP_DATA = pd.read_excel(
+    "static/source/local/Digital Exclusion sources.xlsx",
```

Comment:
The `SOURCE_DATA_FOLDER` should be used here instead

Reply:
Oh yeah sorry! That's correct. It's been run, and the data is included. This script is here just for transparency. The local data file is on my pc.

Reply:
this is what I thought. Thanks